spec/models/miq_server_spec.rb

```diff
@@ -19,7 +19,7 @@
     let(:guid_file) { Rails.root.join("GUID") }
 
     it "should return the GUID from the file" do
-      MiqServer.my_guid_cache = nil
+      MiqServer.my_guid = nil
```

Comment:
To be clear, for future me, the unsafe access,  before this change, was that we synchronize around the reads of this value but don't synchronize the writes.

Reply:
Well if you use the `cattr_accessor` it wasn't synchronized at all.